backend/requirements.txt

```
@@ -22,7 +22,7 @@ pylint==2.4.4
 black
 pytest
 pytest-env
-faker>=2.0,<3.0
+faker>=2.0,<4.0
```

```suggestion
faker>=3.0,<4.0
```

2.0 to 4.0 seems like a larger-than-necessary spread of versions